### DISCUSSION TITLE
How are the resource requiremnts of arcadedb?

### DISCUSSION THREAD

**Discussion by sep2** [ArcadeData/arcadedb#639] [Q&A]
We are evaluting a database choice between `arcadedb`, `neo4j` and `postgres+age` for a new product.

The CPU/memory requirements are main concerns, since we will be deploying the product on small devices.

Do you have any suggestion on this? Thank you.

**Comment by lvca** [comment]
ArcadeDB is pretty flexible. It can run on a large cluster but also on a microPC with only 32MB of heap (not GB, I meant MB). If you do not have a lot of RAM, I suggest you set this at server startup `-Darcadedb.profile=low-ram`.
